Drv/LinuxI2CDriver/old-test/mod.mk

```diff
@@ -0,0 +1 @@
+SUBDIRS = ut
```

Comment:
Is `old-test` supposed to be checked in or has it been replaced by `test`?

Reply:
Meant to be removed, I will delete it and update the branch

Reply:
Thanks.